test/library/standard/CommDiagnostics/doc-examples/CommDiagnosticsExamples.chpl

```diff
@@ -0,0 +1,37 @@
+//
+// THIS TEST CASE IS AUTO-INCLUDED IN THE DOCUMENTATION
+//
+
+use CommDiagnostics;
+
+/* START_EXAMPLE_0 */
+startVerboseComm();
+// between start/stop calls, report comm ops initiated on any locale
+stopVerboseComm();
+/* STOP_EXAMPLE_0 */
+
+/* START_EXAMPLE_1 */
+startVerboseCommHere();
+// between start/stop calls, report comm ops initiated on this locale
+stopVerboseCommHere();
+/* STOP_EXAMPLE_1 */
+
+/* START_EXAMPLE_2 */
+// (optional) if we counted previously, reset the counters to zero
+resetCommDiagnostics();
```

Comment:
Should there be a `resetCommDiagnostics` call in between `STOP_EXAMPLE_0` and `START_EXAMPLE_1`?  Or is that unnecessary since we did a `Here` call instead?